src/components/ClassInfoCard.tsx

```diff
@@ -100,16 +105,36 @@ export const ClassInfoCard: React.FC<ClassInfoProps> = ({
                             {me && me.role === roles.VOLUNTEER
                                 ? cardInfo.volunteerSpaceAvailable +
                                   " volunteer spot" +
-                                  (cardInfo.volunteerSpaceAvailable > 1
+                                  (cardInfo.volunteerSpaceAvailable != 1
```

Comment:
^^ this is unrelated to my ticket but was just changing it so it was plural for 0 as well